flexmeasures/ui/templates/crud/asset.html

```diff
@@ -369,6 +483,438 @@ <h3>All child assets for {{ asset.name }}</h3>
+                            <i class="fa fa-times" 
+                                onclick="removeSensorFromGraph(${index}, ${sensorIndex})"
+                                data-bs-toggle="tooltip" 
+                                data-bs-placement="top" 
+                                title="Remove Sensor"
+                                style="cursor: pointer;"
+                            >
+                            </i>
+
+                            <div style="padding-top: 1px;"></div>
+
+                            <b>Asset:</b> ${Asset.name},
+                            <b>Account:</b> ${Account?.name ? Account.name : "PUBLIC"}
+                        </span>
+                    </div>
+                    `;
+                })
+            );
+
+            const uniqueUnits = [...new Set(sensorsUnits)];
+
+            col.innerHTML = `
+                <div class="card m-0 p-1">
+                    <div class="card-body card-highlight ${newItem.title === selectedGraphTitle ? "border-on-click" : ""}" id="graph_${index}" onclick="selectGraph(${index})">
+                        <div class="d-flex align-items-center">
+                            <div>
+                                ${editingIndex === index
+                                    ? `<input type="text" class="form-control mb-2 me-2" id="editTitle_${index}" value="${newItem.title}" onkeydown="handleEnterKeyEvent(event, ${index})" />`
+                                    : `<h5 class="card-title me-2">${newItem.title}</h5>`
+                                }
+                            </div>
+
+                            <div>
+                                ${editingIndex === index
+                                    ? `<button class="btn btn-success btn-sm ms-2" onclick="saveGraphTitle(${index})">Save</button>`
+                                    : `<button class="btn btn-warning btn-sm ms-2" onclick="editGraphTitle(${index})">Edit</button>`
+                                }
+                            </div>
+                        </div>
+                        <h5 class="card-title pt-2"><b> Sensors: </b></h5>
+                        <div>
+                            ${sensorsContent.length > 0 ? sensorsContent.join("") : "<i>Add sensors on the right side</i>"}
+                            ${uniqueUnits.length > 1 ? `<div class="alert alert-warning" role="alert">Note that you are showing sensors with different units</div>` : ""}
+                        </div>
+                        
+                        <button class="btn btn-danger btn-sm me-2" onclick="removeGraph(${index})">Remove</button>
+                        <button class="btn btn-secondary btn-sm me-2" onclick="moveGraphUp(${index})" ${index === 0 ? "disabled" : ""}>Move Up</button>
+                        <button class="btn btn-secondary btn-sm" onclick="moveGraphDown(${index})" ${index === data.length - 1 ? "disabled" : ""}>Move Down</button>
```

Comment:
This does not work perfectly yet.

- I have three graphs.
- The second graph cannot move down, even if it is not last.
- I move the first downwards (to 2nd position), then it has the same problem.

Reply:
Could you try this again, may there was a bug introduced in previous commit, cause I am not having this issue on my end